06-building-a-program-start-to-finish/solving-the-problem/output/inverted-index.js

```diff
@@ -0,0 +1,298 @@
+/**
```

Comment:
Your `InvertedIndex` class should contain the functions that are related to the domain. It will also store the index on an instance variable of the class and other related data.

It however should not handle things like `localStorage`... and other external DOM functions.